api/institutions/serializers.py

```diff
@@ -0,0 +1,34 @@
+from rest_framework import serializers as ser
+
+from api.base.serializers import JSONAPISerializer, RelationshipField
+
+class InstitutionSerializer(JSONAPISerializer):
+    name = ser.CharField(required=True)
+    id = ser.CharField(required=True, source='_id')
+
+    nodes = RelationshipField(
+        related_view='institutions:institution-nodes',
+        related_view_kwargs={'institution_id': '<pk>'},
+    )
+
+    users = RelationshipField(
+        related_view='institutions:institution-users',
+        related_view_kwargs={'institution_id': '<pk>'}
+    )
```

Comment:
You need `self` and `html` links for Institutions.  You can use `LinksField` for this.